apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -87,6 +100,23 @@ export function getUnpluggedLessonsForScript(state) {
   return _.map(unpluggedStages, filterStageData);
 }
 
+export function getStudentLevelScores(scriptId, sectionId) {
```

Comment:
please call it `fetchStudentLevelScores`, so we know it is being fetched from the server and not just retrieved from redux.